setup.py

```diff
@@ -7,8 +7,8 @@
 PACKAGE_VERSION = '0.2'
 
 deps = [
-    'marionette-client >= 0.12',
-    'marionette-driver >= 0.7',
+    'marionette-client >= 0.15',
+    'marionette-driver >= 0.8',
```

Comment:
Versions have been updated. Please use the latest driver 0.9, what I also wanted to update in a bit.